README.md

```diff
@@ -1,6 +1,6 @@
 # Data Migration Platform
 
-[![Build Status](https://internal.pingcap.net/idc-jenkins/job/build_dm_master/badge/icon)](https://internal.pingcap.net/idc-jenkins/job/build_dm_master/)
+[![Build Status](https://internal.pingcap.net/idc-jenkins/job/build_dm_multi_branch/job/master/badge/icon)
```

Comment:
keep `https://internal.pingcap.net/idc-jenkins/job/build_dm_multi_branch/job/master` as the target link when clicking?

Reply:
done, please check https://github.com/pingcap/dm/pull/904/files